### PR TITLE
completed assessment

### PR DESCRIPTION
**Pull request by wasegree**
# Submitting Week 3 Assessment

### ✅ Completed Files
Mark the following as confirmation that the files in this repository are complete.
- [✅] Coding Challenges
- [✅] Interview Questions
- [✅] React Debugging

### 📝 Feedback
If you would like specific feedback on a particular problem please leave details here:

### CODE REVIEW

File: code-challenges.test.js [hunk 1]
```diff
@@ -53,4 +138,42 @@ const numbersToAdd3 = []
...
+        accumulatingSum += arrayOfNums[i]
+        newArray.push(accumulatingSum)
+      }
+      return newArray
+    
+}
+// Pass
+// console.log(sumArrayNums(numbersToAdd1)) // [2, 6, 51, 60]
+// console.log(sumArrayNums(numbersToAdd2)) // [0, 7, -1, 11]
+// console.log(sumArrayNums(numbersToAdd3)) // []
```
Comment (triciasykes) [inline code-challenges.test.js:179]: William! This is awesome!   I am so proud of you!  You took the feedback and now have created beautiful dynamic functions.  They are very clean.  Your tests are on point, also.  You seem to really understand dynamic functions.  To stretch yourself, try doing number 3 using the reduce method, or test for any edge cases you come across. You are doing great!

File: interview-questions.md [hunk 2]
```diff
@@ -9,59 +9,58 @@ Then, research the question to expand on your answer. Even if you feel you have
...
+2. Object-oriented programming: A way of creating logic/behavior using only objects 
 
-3. Ruby:
+3. Ruby: Ruby is a programming language that uses only objects to perform logic.
 
-4. Ruby blocks:
+4. Ruby blocks: Anonymous functions used in methods.
 
-5. Ruby hashes:
+5. Ruby hashes: a collection of key values that acts similar to an array
```
Comment (triciasykes) [inline interview-questions.md:66]: Your answers are solid. I encourage you to make them more conversational.  The more you can practice describing and explaining these terms and meanings, the better you will appear in an interview. Try adding in examples to show a deep level of understanding.  Great job!

File: react-debugging.html [hunk 3]
```diff
@@ -63,22 +63,27 @@
...
+              <div style={handStyles}>
+                {this.state.hand.map((card, index) => {
+                  return (
+                    <div style={cardStyles} key={index}>
+                      {card}
+                    </div>
+                  )
+                })}
+              </div>
+            </>
```
Comment (triciasykes) [inline react-debugging.html:86]: You nailed it!

**Review by triciasykes — APPROVED** [review]
William, you crushed this assessment!  You have definitely leveled-up this week!  I'm so excited to see your dynamic functions.  Your logic is solid.  I encourage you to expand on your  interview question answers.  Overall, fantastic improvement!  See comments in the files. :)
